index.md

```diff
@@ -6,17 +6,17 @@ weight: 0
 ![Web interface screenshot]({{ site.github.url }}/public/img/web-screenshot.png)
 
 Zipkin is a distributed tracing system. It helps gather timing data needed to
-troubleshoot latency problems in microservice architectures. It manages both the
+troubleshoot latency problems in micro-service architectures. It manages both the
 collection and lookup of this data through a Collector and a Query service.
```

Comment:
for example, it is currently possible to perform this without a process named collector or query